common/verification_system.go

```diff
@@ -0,0 +1,11 @@
+package common
+
+type SystemVerificationId uint16
+
+// TODO Set corrects verification systems
+const (
+	LambdaworksCairo SystemVerificationId = iota
+	GnarkPlonkBls12_381
+	Kimchi
+	Sp1BabyBearBlake3
+)
```

Comment:
The only supported system in this version is Gnark Plonk

Reply:
Done